python/ray/train/lightning/lightning_trainer.py

```diff
@@ -541,7 +541,7 @@ def _lightning_train_loop_per_worker(config):
 
     # Restore from a previously failed run
     checkpoint = session.get_checkpoint()
-    if checkpoint and "ckpt_path" not in trainer_fit_params:
+    if checkpoint:
```

Comment:
we basically ignores what is in ckpt_path and always take checkpoint?
should we add a warning message here before we overwrite the config?
actually why would somebody specify both in this case?

Reply:
No, we only ignore the `ckpt_path` when we are restoring from a failed run(`session.get_checkpoint() is not None`). If we start from scratch, we will still start from `ckpt_path`.

The use case is:
if a user has a checkpoint, and they want to resume training from that checkpoint, and they failed during training and wish to resume training from the latest one. Then we should ignore `ckpt_path` after restoration.

Reply:
yeah, that makes a lot of sense.
can we print a message here though, so we know this is happening in the logs.